src/plugins/i18n.js

```diff
@@ -24,6 +24,9 @@ const messages = {
                 wrongPassword: 'Uneta šifra nije ispravna',
             },
         },
+        addRecipe: {
```

Comment:
Dodaj molim te ovde sve stvari. Nešto će nam možda promaći, možda neki error ili tako nešto ako ih ne dodamo odmah čim napravimo...

Reply:
oću oću samo polako